responsive.css

```diff
@@ -0,0 +1,367 @@
+@media only screen and (min-width: 1025px) {
+  .nav2 {
+    background-color: #fff;
+    color: #aaa;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    font-weight: 500;
+    font-size: 15px;
+  }
+
+  .nav1 {
+    background-color: #131313;
+    color: #9f9f9f;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    font-size: 12px;
+  }
+
+  .mobile-nav2 {
+    display: none;
+    background: rgb(210, 18, 137);
+    background:
+      linear-gradient(
+        45deg,
+        rgba(210, 18, 137, 1) 0%,
+        rgba(255, 67, 34, 1) 100%
+      );
+    width: 60px;
+  }
+
+  .article-container {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    margin: 0 auto;
+    grid-gap: 20px;
+    width: 85%;
+  }
+
+  .news-pages {
+    margin: 25px 50px;
+  }
+
+  .news-container {
+    max-width: 300px;
+    margin: 0 auto;
+  }
+
+  .latest-news {
+    margin: 20px auto;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-gap: 15px;
+  }
+
+  .article2 {
+    display: flex;
+    justify-content: space-between;
+    margin: 100px auto 0 auto;
+    width: 90%;
+  }
+
+  .more-list {
+    margin: 25px 50px;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 20px;
+  }
+
+  .more-list .header-img {
+    width: 100%;
+  }
+
+  .footer2 {
+    display: flex;
+    justify-content: center;
+    background-color: #000;
+    color: #9d9999;
+    padding: 50px;
+  }
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.color-x{
  color: #9d9999;
}
```
**Html file**

```
<div class="color-x">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_